README.md

```diff
@@ -100,6 +100,17 @@ sc.exe create SqlExporterSvc binPath= "%SQL_EXPORTER_PATH%\sql_exporter.exe --co
 `%SQL_EXPORTER_PATH%` is a path to the SQL Exporter binary executable. This document assumes that configuration files
 are in the same location.
 
+## Helm installation
+
+A Helm chart is available for SQL Exporter. It can be installed by running the following commands:
+
+**TL;DR**
+```shell
+helm repo add burningalchemist https://burningalchemist.github.io
+helm intall burningalchemist/sql-exporter
```

Comment:
I've created a similar documentation at artifacthub.io - https://artifacthub.io/packages/helm/sql-exporter/sql-exporter#installing-the-chart. Let's put similar values everywhere just for consistency. There's a small typo, besides. 👍

Reply:
Just a small reminder to address this little issue here. 👍